packaging/linux/run_keybase

```diff
@@ -196,19 +196,25 @@ start_background() {
   if [ "$KEYBASE_NO_KBFS" != "1" ]; then
       run_redirector_in_background
       echo Starting KBFS...
-      kbfsfuse -debug -log-to-file &>> "$logdir/keybase.start.log" &
+
+      # The only time kbfsfuse -log-to-file prints to stdout is if the mount
+      # fails. So, if it does fail, because the directory does not exist, or
+      # has invalid permissions, the user will be notified on the command line
+      # after running `run_keybase`, but otherwise stdout will not be cluttered
+      # with other various log messages.
+      ( kbfsfuse -debug -log-to-file | tee "$logdir/keybase.start.log" ) &
```

Comment:
This isn't the greatest because it's a background process printing to stdout which is annoying but should be fine because it's only in the error case and it's just one line.